prisma/schema/campaign.jsonTypes.ts

```diff
@@ -22,24 +22,48 @@ declare global {
       pledged?: boolean
       customIssues?: Record<'title' | 'position', string>[]
       runningAgainst?: Record<'name' | 'party' | 'description', string>[]
-      party?: string //TODO: enumerate all parties we want to allow?
-      otherParty?: string
-      office?: string
+      geoLocation?: {
+        geoHash?: string
+        lng?: number
+        lat?: number
+      }
+      geoLocationFailed?: boolean
+      city?: string | null
+      county?: string | null
+      normalizedOffice?: string | null
       otherOffice?: string
-      website?: string
+      office?: string
+      party?: string
+      otherParty?: string
       district?: string
+      raceId?: string
+      noNormalizedOffice?: boolean
+      website?: string
       pastExperience?: string
       occupation?: string
       funFact?: string
       campaignCommittee?: string
       statementName?: string
     }
-
     export type CampaignData = {
       createdBy?: 'admin' | string
-      launchStatus?: CampaignLaunchStatus
+      hubSpotUpdates?: {
+        verified_candidates?: string
+        election_results?: string
+        office_type?: string
+      }
+      slug?: string | null
       currentStep?: OnboardingStep
-      slug?: string
+      launchStatus?: CampaignLaunchStatus
+      customIssues?: Record<'title' | 'position', string>[]
+      runningAgainst?: Record<'name' | 'party' | 'description', string>[]
+      website?: string
+      district?: string
+      pastExperience?: string
+      occupation?: string
+      funFact?: string
+      campaignCommittee?: string
+      statementName?: string
```

Comment:
There's still a lot of fields on here that are not on `.data` but on `.details` in the current app.